lib/pharos/addon_manager.rb

```diff
@@ -10,11 +10,19 @@ class AddonManager
     class InvalidConfig < Pharos::Error; end
     class UnknownAddon < Pharos::Error; end
 
+    # @return [Array<Pharos::Addon>]
+    def self.addons
+      @addons ||= []
+    end
+
     # @param dirs [Array<String>]
+    # @return [Array<Pharos::Addon>]
     def self.load_addons(*dirs)
       dirs.each do |dir|
-        Dir.glob(File.join(dir, '**', '*.rb')).each { |f| require(f) }
+        Dir.glob(File.join(dir, '**', 'addon.rb')).each { |f| require(f) }
```

Comment:
I dislike this `addon.rb` thing a little because now when editing multiple files, all the tabs will just say "addon.rb". Also makes any fancier file fuzzyfinding thing annoying, you can only complete up to the dirname.

Reply:
Fixed name makes it more consistent? If filename can be anything it's a bit misleading signal for users.. can they have multiple `.rb` files within a same folder etc...

Reply:
Yep, probably simpler.